src/model/Client.ts

```diff
@@ -7,10 +7,12 @@ import { DbMappingConstants as DB } from '../database/DbMappingConstants';
  */
 export class Client extends Entity {
     adminUsers: Array<string>;
+    allowedUrls: Array<string>;
 
-    constructor(clientId: string, adminUsers: Array<string>) {
-        super(clientId, clientId);
```

Comment:
Przez to nie dzialal `modify`.

w konstruktorze `Entity` pole `client` jest snippowane, a `entity` nie.